tools/topology/sof-tgl-max98357a-rt5682.m4

```diff
@@ -85,8 +85,6 @@ define(DMIC_PIPELINE_KWD_ID, `12')
 define(DMIC_DAI_LINK_16k_ID, `2')
 # define pcm, pipeline and dai id
 define(KWD_PIPE_SCH_DEADLINE_US, 5000)
-# Use the google hotword detector
-define(DETECTOR_TYPE, `google-hotword-detect')
```

Comment:
this file and the two below are the only ones that need to reverted, the rest are fine

Reply:
feel free to suggest an alternate solution with a better PR @cujomalainey, I just wanted to restore functionality in our tests without delay. The more we delay fixes generally the more difficult it is to root cause new issues and unrelated problems

Reply:
See #4358 4358